docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -97,3 +103,4 @@ If you have any questions regarding implementing this exercise, please post them
 
 [docs-concept-exercises]: ../concept-exercises.md
 [reference]: ../../reference/concepts/README.md
+[meta-design]: https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/enums-advanced/.meta/design.md
```

Comment:
Perhaps this should be a relative link so that it works offline and on the website.

Reply:
Excellent point. Updated.